<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>437</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="file_edit"/>
          </item>
          <item>
           <widget class="QPushButton" name="select_button">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="run_button">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="args">
            <property name="text">
             <string>Arguments:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="args_edit"/>
          </item>
          <item>
           <widget class="QPushButton" name="stop_button">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="out">
          <property name="text">
           <string>Out</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="console_edit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="exit_code">
          <property name="text">
           <string>Exit code</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="exit_code_edit"/>
        </item>
        <item>
         <widget class="QLabel" name="duration">
          <property name="text">
           <string>Duration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="duration_edit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
